clang-tools-extra/clang-tidy/daedalean/PreprocessingDirectivesCheck.cpp

```diff
@@ -23,21 +23,22 @@ namespace {
 class PreprocessingDirectivesPPCallbacks : public PPCallbacks {
 public:
   explicit PreprocessingDirectivesPPCallbacks(ClangTidyCheck &Check,
-                                   const SourceManager &SM)
+                                              const SourceManager &SM)
       : Check(Check), SM(SM) {}
 
   virtual void PragmaDirective(SourceLocation Loc,
                                PragmaIntroducerKind Introducer) override {
     const auto fileId = SM.getFileID(Loc);
     if (!fileId.isValid()) {
-        return;
+      return;
     }
     const auto fileEntry = SM.getFileEntryForID(fileId);
     if (!fileEntry) {
-        return;
+      return;
     }
     const auto fileName = fileEntry->getName();
-    if (fileName.endswith(".hh") || fileName.endswith(".h")) {
+    if (fileName.endswith(".hh") || fileName.endswith(".h") ||
+        fileName.endswith(".hpp")) {
```

Comment:
We do not have hpp files, bug is in test filename

Reply:
Then the test won't run, because it is not an expected extension from llvm...

Reply:
Fixed to run tests with .hh